src/components/Feilresponser/FeilresponserUtils.ts

```diff
@@ -0,0 +1,150 @@
+import { ErrorMessage } from '~api/apiClient';
+import { MessageFormatter } from '~lib/i18n';
+import { Nullable } from '~lib/types';
+
+import feilresponseMessages from './feilresponser-nb';
+import { revurderingFeilkodeTilFeilmelding } from './revurderingFeilresponser/RevurderingFeilresponser';
+import revurderingMessages from './revurderingFeilresponser/RevurderingFeilresponser-nb';
+import { søknadsbehandlingFeilkodeTilFeilmelding } from './søknadsbehandlingFeilresponser/SøknadsbehandlingFeilresponser';
+import søknadsbehandlingMessages from './søknadsbehandlingFeilresponser/søknadsbehandlingFeilresponser-nb';
+
+const ukjentFeil = 'Ukjent feil';
+
+export const feilresponseTilFeilmelding = (
+    formatMessage: MessageFormatter<
+        typeof revurderingMessages | typeof søknadsbehandlingMessages | typeof feilresponseMessages
+    >,
+    feil?: Nullable<ErrorMessage>
+) => {
+    const søknadsbehandlingError = søknadsbehandlingFeilkodeTilFeilmelding(formatMessage, feil);
+    if (søknadsbehandlingError !== ukjentFeil) {
+        return søknadsbehandlingError;
+    }
```

Comment:
Det føles litt skjørt å skulle sjekke på den eksakte strengen. Kunne vi heller ha endret sånn at `søknadsbehandlingFeilkodeTilFeilmelding` og `revurderingFeilkodeTilFeilmelding` returnerer `null` dersom de ikke har den?

Reply:
Hvis disse skal returnere null, så må det bli håndtert av utsiden. Det er kanskje ikke et problem, siden jeg ser for meg at parent komponenten skal bli brukt i 95% av alle tilfellene.